scripts/ci-e2e.sh

```diff
@@ -41,6 +41,13 @@ source "${REPO_ROOT}/hack/ensure-kustomize.sh"
 ARTIFACTS="${ARTIFACTS:-${PWD}/_artifacts}"
 mkdir -p "$ARTIFACTS/logs/"
 
+# our exit handler (trap)
+cleanup() {
+  # stop boskos heartbeat
```

Comment:
Want to move anything else in here?

Reply:
I thought about moving the account checkin code there, but the worst that happens is that we have to wait for the reaper to reclaim if it isn't called. I think killing the heartbeat is good enough for now, since it would prevent issues for someone manually running it where the background process may live on.